src/main.ts

```diff
@@ -77,13 +78,15 @@ async function run() {
       const client: github.GitHub = new github.GitHub(GITHUB_TOKEN);
       addLabels(client, labelData);
 
-      const prData: PullsUpdateParams = {
-        owner: organization.login,
-        repo,
-        pull_number: prNumber,
-        body: getPrDescription(prBody, story),
-      };
-      updatePrDetails(client, prData);
+      if (shouldUpdatePRDescription(prBody)) {
+        const prData: PullsUpdateParams = {
+          owner: organization.login,
+          repo,
+          pull_number: prNumber,
+          body: getPrDescription(prBody, story),
+        };
+        await updatePrDetails(client, prData);
```

Comment:
do we need an `await` if we already using an `await` inside the method?

Reply:
IIRC async functions return promises right? else the code after it will run before the internal awaits..

Reply:
yes it does return promise. Can you please add to the other async call as well where we addLabel

Reply:
we were just calling the methods and not waiting for the result. and btw if we are not utilizing the response do we need an await?

Reply:
I agree that it returns a promise. this is like fire and forget call. :thinking:

Reply:
added await for `addLabels` as well